src/helpers/FormioScript/FormioScript.stories.mdx

```diff
@@ -4,9 +4,16 @@ import { Meta } from "@storybook/addon-docs";
 
 # FormioScript
 
-FormioScript provide shorthand implementation that helps loading the necessary scripts that need to load a Formio form in your application.
+FormioScript provide shorthand implementation to lazy load the necessary libraries and scripts that need to initiate a Formio form in your application.
```

Comment:
The grammar here could use more work. Perhaps:

"FormioScript provides a shorthand implementation to lazy-load the necessary libraries and scripts to initiate a Formio form in your application."